common/src/main/java/org/ergoplatform/appkit/SecretString.java

```diff
@@ -93,4 +93,8 @@ public boolean equals(Object obj) {
     public String toStringUnsecure() {
         return String.valueOf(_data);
     }
+
+    public org.ergoplatform.wallet.interface4j.SecretString toInterface4JSecretString() {
+        return org.ergoplatform.wallet.interface4j.SecretString.create(_data.clone());
+    }
```

Comment:
Hm, I think we don't need two implementations of SecretString. 
Any reason we cannot just use org.ergoplatform.wallet.interface4j.SecretString everywhere?

Reply:
As noted, this would be a breaking change because of the package change and some methods missing. I don't know why the class wasn't adopted unchanged. If we do a breaking change, next version should be 4.1.x or 5.0.x and not 4.0.6. But I think after all this time, a non-breaking bugfix release is necessary.